src/trusted-types.ts

```diff
@@ -0,0 +1,17 @@
+interface CSPTrustedHTMLToStringable {
+  toString: () => string
+}
+
+interface CSPTrustedTypesPolicy {
+  createHTML: (s: string) => CSPTrustedHTMLToStringable
+}
+
+let cspTrustedTypesPolicy: CSPTrustedTypesPolicy | null = null
+
+export function getCSPTrustedTypesPolicy() {
+  return cspTrustedTypesPolicy
+}
+
+export function setCSPTrustedTypesPolicy(policy: CSPTrustedTypesPolicy | null) {
```

Comment:
Similar to my comments in `include-fragment`, I wonder if it would be more clear of the intent if this were attached to the `TemplateResult` class as a static member, so the invocation would be:


```js
import {TemplateResult} from '@github/jtml'
TemplateResult.setCSPTrustedTypesPolicy(policy)
```

Reply:
I had considered that, however there's one call to `innerHtml` outside the scope of the `TemplateResult` class in the `unsafeHtml` code, which doesn't live in a class. https://github.com/github/jtml/blob/2208fa1b6138b590288f4cfdf2aaa20c210568dc/src/unsafe-html.ts#L8

I could import `TemplateResult` into the `unsafeHtml` file, however I'm not sure if that makes the intent clear?

Reply:
I think it's okay for that function to depend on `TemplateResult`, as it already implicitly depends on other parts of the module, so the association feels clear.

My main motivation for attaching these as static to classes is because I imagine in a codebase which pulls them all in (ie github), they will all be centralised in one file, and therefore we need to make it evident onto which modules are we attaching which policies (or even if they all share a single policy). Consider:

```js
import IncludeFragmentElement from '@github/include-fragment-element'
import {TemplateResult} from '@github/jtml'
const policy = ....

IncludeFragmentElement.setCSPTrustedTypesPolicy(policy)
TemplateResult.setCSPTrustedTypesPolicy(policy)
```

compare this to functions with the same name and we have to do this awkward dance:

```js
import {setCSPTrustedTypesPolicy as IncludeFragmentsetCSPTrustedTypesPolicy} from '@github/include-fragment-element'
import {setCSPTrustedTypesPolicy as JTMLsetCSPTrustedTypesPolicy} from '@github/jtml'
const policy = ....

IncludeFragmentsetCSPTrustedTypesPolicy(policy)
JTMLsetCSPTrustedTypesPolicy(policy)
```